GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest

```diff
@@ -13,10 +13,10 @@
         <Tags>Unit Test, C++, Google Test, GoogleTest, GTest, Test Explorer, Testing</Tags>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Ultimate" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Pro" />
```

Comment:
Is the new version built still going to be compatible with Dev15?

Reply:
Good call, doesn't look like it's compatible with pre-17. I'll update this.